modules/learn/pages/clusters-and-availability/rebalance.adoc

```diff
@@ -142,62 +142,63 @@ The effects on services other than the Data Service are described below.
 
-If more index replicas exist than can be handled by the number of existing nodes, replicas are dropped: the numbers are automatically made up subsequently, if additional Index Service nodes are added to the cluster.
+Rebalance when you add and remove index nodes::
+A swap rebalance moves indexes from nodes you're removing and places them on the nodes you're adding.
 
-During rebalance, no index node is removed until index-building has completed on alternative nodes.
-This ensures uninterrupted access to indexes.
+You can have a rebalance redistribute indexes im more cases by enabling the **Optimize Index Placement On Rebalance** setting on the Couchbase Server Web Console's xref:manage:manage-settings/general-settings.adoc#index-storage-mode[Settings page]. 
+You can also change this setting using the xref:manage:manage-settings/general-settings.adoc#index-settings-via-rest[Settings REST API] where it is named `redistributeIndexes`.
 
-==== Smart Batching
+NOTE: In Couchbase Server 7.2 and later, the redistribution setting affects both partitioned and non-partitioned indexes.
 
-When rebalance occurs, indexes are relocated and rebuilt in _batches_, so as to place a limit on the amount of work that is attempted concurrently -- the default batch size being `3`.
+Enabling the index redistribution setting causes a rebalance to redistribute indexes in the following situations:
 
-In Couchbase Server Version 7.1 and after, _smart batching_ is used; so as to reduce further the time and resources required to move index metadata, and to rebuild indexes at their new locations during rebalance.
-Specifically, smart batching:
+Rebalance after you add an index node::
+Rebalance optimizes index placement across _all_ index nodes in the cluster, including on the new index nodes.
 
-* Increases rebalance performance by increasing pipeline parallelism, through starting the next batch while the prior batch is only partially completed.
+Rebalance after you add or remove a non-index node::
+Rebalance moves indexes from heavily loaded nodes to nodes with free resources to balance distribution.
 
-* Allows overall concurrency to be increased, through administrator-determined modifications of the batch size.
+Rebalance during an index server group repair::
+A group failure in a multiple server group database can force all replicas into a single group. In this case, rebalance redistributes the replicas to support high availability across server groups after the server group repair.
 
-* Optimizes the sharing of data streams, whereby index relocation occurs.
+NOTE: If after you drop Index Service nodes, the remaining nodes cannot handle all of the index replicas, Couchbase Server drops some of the replicas. If you later add additional Index Service nodes to the cluster, Couchbase Server replaces the dropped replicas.
 
-* Repairs, prior to index relocation, index replicas lost due to node failure.
+==== Index Rebalance Batch Size
```

Comment:
The batch size works only if fast rebalance option is not selected. For fast rebalance, the transfers are managed by the server and we don't think those controls need to be exposed to the users.

Reply:
Changed to reflect that this only applies to rebuilding the indexes, Also updated the REST API page to remove references to batching index moves together.